src/scripts.js

```diff
@@ -235,7 +235,7 @@ const showAllRecipes = () => {
     allRecipesView.insertAdjacentHTML('afterbegin', `
     <article class="recipes-views" id= ${recipe.id}>
     <h1>${recipe.name}</h1>
-    <img src="${recipe.image}">
+    <img src="${recipe.image}" tabindex= "0">
```

Comment:
This is a super cool way to add tabbing to an element being pushed into the DOM.